test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -271,6 +271,89 @@ function ExecuteRequestWithCustomUserAgent {
     return $result
 }
 
+function GetSelfSignedCert {
```

Comment:
Move the module to a common location and add your function to it, with descriptions of each cert (at least what you learned.)  At the minimum, file an issue that this work needs to be done to avoid future duplication of effort.  Even then, I would suggest adding the comments to each now.